buildroot.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
+is_arg=false
 
-docker-compose pull
-docker-compose run --rm buildroot
+for args in "$@"
+do
+  is_arg=true
 
+  if [ "$args" = "--help" ]; then
+    echo "Buildroot help..."
+    echo "  --help          Displays this help message"
+    echo "  --clean         Stops the buildroot container, removes the container and its data, pulls the most recent version, and updates the container with the most recent image"
+    echo "  --setup         Setup environment dependancies"
+  elif [ "$args" = "--clean" ]; then
+    echo "Cleaning system..."
+    docker-compose stop
+    docker-compose rm -f
+    docker-compose pull
+    docker-compose up -d
+  elif [ "$args" = "--setup" ]; then
+    echo "Setting up system..."
+    echo "This assumes that you already have docker, docker-compose, and git already installed"
+    git submodule update --init --recursive
```

Comment:
I don't think this is necessary. Why is this easier than just running the command directly?